app/assets/stylesheets/styles/guides/_new_guides.css.scss

```diff
@@ -479,8 +479,35 @@
 
     /* each step has actions to fill out */
     .actions {
```

Comment:
Hmm, I think the purpose of that was that they'd be reusable on #show and #new but maybe I messed up; anyway they aren't used on #show anymore.